content/en/docs/ops/diagnostic-tools/multicluster/index.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Troubleshooting Multi-Cluster
+description: Describes tools and techniques to diagnose issues with multi-cluster and multi-network installations.
+weight: 20
```

Comment:
The weight collides with two other pages (we should update one of them as well): proxy-command and virtual-machines. The weights should be set up so the items on the diagnostic-tools page show in the desired order.